pages/search.js

```diff
@@ -282,6 +282,7 @@ class SearchPage extends React.Component {
     const showSortFilterSection = accounts?.nodes?.length > 0;
     const getSortOption = value => ({ label: i18nSearchSortingOptions(intl, value), value });
     const sortOptions = [getSortOption('ACTIVITY'), getSortOption('CREATED_AT.DESC'), getSortOption('CREATED_AT.ASC')];
+    const selectedTypeFilter = router.query.isHost ? 'HOST' : router.query.type || 'ALL';
```

Comment:
You're doing the same mistake as the other day and gonna break server side rendering. 

Why use `router` when you have `props`?

Reply:
Ops. I've corrected this. I've also noticed that we have this same problem with the country filter as well as the search bar within the `/search` page. I've added some changes to update all the filters based on the url back/forward navigation. 🤔